tools/deployment_manager/configs/linux.json

```diff
@@ -72,7 +72,7 @@
       "files": [
         "runtime/lib/intel64/libgna.so",
         "runtime/lib/intel64/libgna.so.2",
-        "runtime/lib/intel64/libgna.so.3.0.0.1377",
+        "runtime/lib/intel64/libgna.so.3.0.0.1455",
```

Comment:
BTW, why do we need all these libraries? I suppose we need to install only `runtime/lib/intel64/libgna.so.2`

```sh
$ readelf -d libov_intel_gna_plugin.so

Dynamic section at offset 0x4bfe60 contains 33 entries:
  Tag        Type                         Name/Value
 0x0000000000000001 (NEEDED)             Shared library: [libov_runtime.so]
 0x0000000000000001 (NEEDED)             Shared library: [libgna.so.2]
 0x0000000000000001 (NEEDED)             Shared library: [libstdc++.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libm.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libgcc_s.so.1]
 0x0000000000000001 (NEEDED)             Shared library: [libpthread.so.0]
 0x0000000000000001 (NEEDED)             Shared library: [libc.so.6]
```

Reply:
While I was using deployment manger which is present in tools directory  , it seems that it required this library for task

Reply:
but why? we can se that only `libgna.so.2` is used

Reply:
I was working on Deployment manager for Some of specific devices including GNA , it was looking for this library

Reply:
[ 2022-01-11 19:48:05,053 ] WARNING : [vpu] Missing /home/aishwarya/openvinomake/usr/local/runtime/3rdparty/97-myriad-usbboot.rules
[ 2022-01-11 19:48:05,053 ] WARNING : [gna] Missing /home/aishwarya/openvinomake/usr/local/runtime/lib/intel64/libgna.so.3.0.0.1377
[ 2022-01-11 19:48:05,053 ] WARNING : [hddl] Missing /home/aishwarya/openvinomake/usr/local/runtime/lib/intel64/libHDDLPlugin.so
[ 2022-01-11 19:48:05,053 ] WARNING : [opencv] Missing /home/aishwarya/openvinomake/usr/local/extras/opencv/version.txt
[ 2022-01-11 19:48:05,053 ] WARNING : [python_common] Missing /home/aishwarya/openvinomake/usr/local/python/python3
[ 2022-01-11 19:48:05,053 ] WARNING : [python3.6] Missing /home/aishwarya/openvinomake/usr/local/python/python3.6
[ 2022-01-11 19:48:05,053 ] WARNING : [python3.7] Missing /home/aishwarya/openvinomake/usr/local/python/python3.7
[ 2022-01-11 19:48:05,053 ] WARNING : [python3.8] Missing /home/aishwarya/openvinomake/usr/local/python/python3.8
[ 2022-01-11 19:48:05,053 ] WARNING : [python3.9] Missing /home/aishwarya/openvinomake/usr/local/python/python3.9
Traceback (most recent call last):
  File "./deployment_manager.py", line 25, in <module>
    sys.exit(main())
  File "/home/aishwarya/openvinomake/usr/local/tools/deployment_manager/deployman/main.py", line 214, in main
    raise ValueError("You input incorrect target. {} is not available.".format(target_name))
ValueError: You input incorrect target. gna is not available.






























This is error message with initial code

Reply:
observing the logs it seems it was looking for this library .